parkhangs-webapp/README.md

```diff
@@ -121,14 +121,14 @@ my-app/
     App.js
     App.test.js
     index.css
-    index.js
+    parkController.js
     logo.svg
 ```
 
 For the project to build, **these files must exist with exact filenames**:
 
 * `public/index.html` is the page template;
-* `src/index.js` is the JavaScript entry point.
```

Comment:
ahh what is happening?? this is the webapp, not the express app!

Reply:
Im not sure why this was changed but ill add it back